.docker/images/govcms8/settings/settings.php

```diff
@@ -240,6 +240,11 @@ public function __toString() {
   }
 }
 
+// Enforce correct solr server configuration (GOVCMS-4634)
+// Fix for 8.5.0 and the solr upgrade.
+$config['search_api.server.lagoon_solr']['backend_config']['connector_config']['path'] = '/';
```

Comment:
The PaaS site will be good from this change. Can someone pls confirm? thanks

Reply:
This only enforces the new path/core for the lagoon_solr service. PaaS should be using this and PaaS can always update this value later in the settings file.

Reply:
@steveworley thanks, Do you think we may have to note this change to the PaaS client and helpdesk as the PaaS may have troubles?

Reply:
Yeah I would say we should include all changes in comms to the relevant parties (regardless of how big/small).